framework/src/org/apache/cordova/ConfigXmlParser.java

```diff
@@ -33,7 +33,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 public class ConfigXmlParser {
     private static String TAG = "ConfigXmlParser";
 
-    private String launchUrl = "file:///android_asset/www/index.html";
+    private String launchUrl = "https://localhost/";
```

Comment:
Do we want to use the hostname and scheme preferences from iOS here?

Reply:
If possible, that would be better I believe.

Reply:
I will try to do this monday or so.

Reply:
I already did it in SystemWebViewClient but the change in `getLaunchUrl` was necessary too.